### DISCUSSION TITLE
Webpack 5 img tag not viewed after npm run

### DISCUSSION THREAD

**Discussion by AqilbekAbilaev** [webpack/webpack#14849] [Q&A]
{
                test: /\.png$/,
                type: 'asset/resource'
            }

           <img src="../img/push.png" alt="hero_image">
            
Here is my webpack file and html tag and I import img in index.js but after bundling img does not show (

**Comment by alexander-akait** [comment]
Check your public path

**Reply by AqilbekAbilaev** [reply]
It is showing before bundling

**Reply by AqilbekAbilaev** [reply]
are my webpack settings true ?

**Reply by AqilbekAbilaev** [reply]
But if I put this img as an background this works 🤦‍♂️  but in html tag it is not showing :(

**Reply by alexander-akait** [reply]
If you don't see image, it means you have invalid path to image, check it
